### ISSUE TITLE
💡[FEATURE]: Enhancing the scroll bar's accessibility and aesthetic.

### ISSUE THREAD

**Issue by sabhansali** [Muskansahuincredible/StudyNotion-An-Online-Education-Platform#241]
### Description

The scroll bar is thin and not accessible or visible easily. I would like to improve the accessibility of the scroll bar and make it visually appealing, under labels GSSoC24 and level of issue. 

### Checklist

- [X] I have checked the existing issues.
- [X] I have read the Contributing Guidelines
- [X] I have read the Code Of Conduct
- [X] The changes don't break the code
